3rdparty/java/BUILD

```diff
@@ -6,90 +6,90 @@ java_library(
     name = "aws-java-sdk",
```

Comment:
Thanks for the fix. In order for the travis ci to compile correctly, you need to download bazel 0.2.3 in the .travis.yml 
https://github.com/twitter/heron/blob/master/.travis.yml

Can you make the changes and update the PR?